setup.py

```diff
@@ -61,7 +61,7 @@ def package_files(directory):
 other capabilities.
 """
 CORE_REQUIREMENTS = SKINNY_REQUIREMENTS + [
-    "alembic<=1.4.1",
+    "alembic!=1.4.1",
```

Comment:
Is some bug in alembic==1.4.1 fixed in greater version ? What's the bug here make us exclude alembic==1.4.1 ?